daemon/money/money.go

```diff
@@ -0,0 +1,15 @@
+package money
+
+import (
+	"github.com/shopspring/decimal"
+)
+
+type Money int64
```

Comment:
Please doc this is sats, also, use `uint64` I dont see negative amounts anywhere involved.

Reply:
Done 🏁